recipes/venvstacks/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "venvstacks" %}
+{% set version = "0.2.0" %}
+{% set python_min = "3.11" %}
```

Comment:
Can already create a `conda_build_config.yaml` which the bot would know about.

```yaml
python_min:
  - "3.11"
```

Reply:
I kind of prefer to have that in the recipe. It is easier to have that info in a single place so we can remember to update it.